bigquery/cloud-client/natality_tutorial.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def run_natality_tutorial():
+    # [START bigquery_query_natality_tutorial]
```

Comment:
Question: do we want to own this as a BigQuery sample or should it go under Dataproc?

I'm leaning towards BigQuery, too, since there doesn't appear to be anything Dataproc-specific here.

Reply:
I put it as BigQuery and didn't include Dataproc in the name because there's nothing Dataproc-related in this sample (except the docstring, which we could move into the docs instead of the sample) and we could potentially leverage this sample for other purposes.